sound/soc/sof/intel/hda-dai-ops.c

```diff
@@ -311,7 +331,7 @@ static int hda_ipc4_post_trigger(struct snd_sof_dev *sdev, struct snd_soc_dai *c
 }
 
 static const struct hda_dai_widget_dma_ops hda_ipc4_dma_ops = {
-	.get_hext_stream = hda_get_hext_stream,
```

Comment:
actually question @ranj063: is this intentional that we don't modify the chain DMA ?

````c
static const struct hda_dai_widget_dma_ops hda_ipc4_chain_dma_ops = {
	.get_hext_stream = hda_get_hext_stream,
````

Reply:
@plbossart yes. Chain DMA does not use the pipelines in topology